src/connectors/README.md

```diff
@@ -165,6 +169,7 @@ Connects provided component with a product context
 - `productId (string)` - Id of the current shown product
 - `variantId (string)` -  Id of the current selected variant
 - `conditioner (Function)` - Helper class for ProductCharacteristic component
+- `state (object)` - Route state passed from history action
```

Comment:
What's the content of the state object? Please describe it in a way that everyone knows what's inside and how to use it.

Reply:
Updated Readme to let user know that state values are defined in route action